util/src/lib.rs

```diff
@@ -156,59 +156,5 @@ pub use log::*;
 pub use kvdb::*;
 pub use timer::*;
 
-#[cfg(test)]
```

Comment:
ahh... you are right. this test was just moved to different file `util/bigint/src/uint.rs', cause it fits there better